docs/build-deploy.md

```
@@ -0,0 +1,81 @@
+| `BREAKING CHANGE:` | Major (2.1→3.0) | API change |
+
+**Config files:** `.release-please-config.json`, `.release-please-manifest.json`
+
+## Deployment Targets
+
+### Web
+- Expo web export to static files
+- Deployed to GitHub Pages or Render
+- URL: `https://arcade-cabinet.github.io/otter-river-rush`
+
+### iOS
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)